docs/domain/id/index.md

```diff
@@ -1,31 +1,25 @@
-# IDs
+# Id
 
-CoreLib supports three different flavors of IDs:
+[Aggregate] requires you to specify identity type. Every aggregate has an `Id` property of the specified type. CoreLib supports three different flavors of IDs:
 
-1. Primitive types like `Guid`, `int`, etc.
-2. Generic type wrappers,
-3. Source generated IDs.
+- Primitive types like `Guid`, `int`, etc.
+- Generic type wrappers.
+- Source generated IDs.
 
 From CoreLib v8, Source Generated IDs are the default one, with primitive types being fallbacks if source generated one cannot be used.
 
-## Types
+## Packages
 
-### Source generated IDs
+| Package | Link | Application in section |
+| --- | ----------- | ----------- |
+| LeanCode.DomainModels | [![NuGet version (LeanCode.DomainModels)](https://img.shields.io/nuget/vpre/LeanCode.DomainModels.svg?style=flat-square&logo=nuget)](https://www.nuget.org/packages/LeanCode.DomainModels/8.0.2260-preview/) | `IAggregateRoot` |
+| LeanCode.DomainModels.Generators | [![NuGet version (LeanCode.DomainModels.Generators)](https://img.shields.io/nuget/vpre/LeanCode.DomainModels.Generators.svg?style=flat-square&logo=nuget)](https://www.nuget.org/packages/LeanCode.DomainModels.Generators/8.0.2260-preview/) | Ids |
 
-Source generated IDs leverage [Source Generators](https://learn.microsoft.com/en-us/dotnet/csharp/roslyn-sdk/source-generators-overview) to generate fully functional ID structs that can work as IDs for entities. They are specialized for a particular entity type, but don't use the generic mechanisms of the language. They support the same feature set as aforementioned IDs.
-
-### Generic type wrappers
-
-The domain part of the library supports a set of generic IDs:
+## Source generated IDs
 
-1. [Id&lt;T&gt;],
-2. [IId&lt;T&gt;],
-3. [LId&lt;T&gt;],
-4. [SId&lt;T&gt;].
-
-All the types give you type safety when passing the IDs, without introducing penalty (they basically work as `newtype`s). Unfortunately, they require an entity to be defined beforehand - it works as a generic parameter. This means you can't use it without a corresponding entity type. This poses a problem if you want to use the ID outside the parent domain. It is also quite hard to use - you need to know the exact ID format before you reference it (you need to choose between the four types when you just want to reference other entity).
+Source generated IDs leverage [Source Generators](https://learn.microsoft.com/en-us/dotnet/csharp/roslyn-sdk/source-generators-overview) to generate fully functional ID structs that can work as IDs for entities. They are specialized for a particular entity type, but don't use the generic mechanisms of the language. They support the same feature set as aforementioned IDs.
```

Comment:
This paragraph assumes that someone used `SId` (et al.) earlier - otherwise, they don't really know what generics are you talking about.